pkg/api/types.go

```diff
@@ -2563,4 +2661,12 @@ type RangeAllocation struct {
 const (
 	// "default-scheduler" is the name of default scheduler.
 	DefaultSchedulerName = "default-scheduler"
+
+	// RequiredDuringScheduling affinity is not symmetric, but there is an implicit PreferredDuringScheduling affinity rule
+	// corresponding to every RequiredDuringScheduling affinity rule.
+	// DefaultHardPodAffinityWeight represents the default weight of implicit PreferredDuringScheduling affinity rule.
```

Comment:
Maybe change the last sentence to "When the --hard-pod-affinity-weight scheduler flag is not specified, DefaultHardPodAffinityWeight defines the weight of the implicit PreferredDuringScheduling affinity rule."